backend/game/turn_actions/attack.py

```diff
@@ -1,19 +1,24 @@
-def is_attackable(attacking_player_location, other_player):
-    if attacking_player_location != other_player.location:
-        return True
-    else:
-        return False
+from game.values.locations import Locations
+from game.turn_actions.player_movement import take_tokyo_on_kill
+
+
+def is_attackable(attacking_player, other_player):
+    if attacking_player.location != other_player.location:
+        if Locations.OUTSIDE in {attacking_player.location, other_player.location}:
```

Comment:
Sight read - I am not sure this is correct python syntax.  Not sure exactly what logic you are trying to do in this statement